src/main/resources/META-INF/rewrite/no-guava.yml

```diff
@@ -70,6 +71,34 @@ recipeList:
       oldFullyQualifiedTypeName: com.google.common.base.Function
       newFullyQualifiedTypeName: java.util.function.Function
 
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.guava.PreferJavaUtilOptional
+displayName: Prefer `java.util.Optional`
+description: Prefer `java.util.Optional` instead of using `com.google.common.base.Optional`.
+tags:
+  - guava, RSPEC-4738
```

Comment:
Multiple tags should be on separate lines. I can see that this was just copy-pasted, so we might as well correct the others.